models/articles.model.js

```diff
@@ -2,21 +2,26 @@ const db = require('../db/connection');
 const format = require('pg-format');
 
 exports.selectArticles = () => {
-	const queryTxt = 'SELECT * FROM articles';
+  const queryTxt = 'SELECT * FROM articles ORDER BY created_at DESC;';
 	return db.query(queryTxt).then((result) => {
 		return result.rows;
 	});
 };
 
 exports.selectArticleById = (article_id) => {
-	const queryTxt = format('SELECT * FROM articles WHERE article_id = $1');
+  const queryTxt = format(`
```

Comment:
I think this is the same as the previous PR? So just get rid of the pg-format here 👍🏼

Reply:
Oh, yeah you are right. I think the previous one includes all of the commits I've made from this one. Thank you for checking out. I'll check PG-format 🙂